src/index.html

```diff
@@ -4,9 +4,338 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img
+            src="./images/logo.png"
+            alt="Photo logo MOYO"
+            class="top-bar__img"
+          >
+        </a>
+
+        <nav class="nav top-bar__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#Apple">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#Samsung">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#Smartphones">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link nav__link--text_color"
+                href="#Laptops_&_computers"
+                data-qa="nav-hover"
+              >
+                Laptops & computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#Gadgets">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#Tablets">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#Photo">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#Video">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="content">
+      <div class="content__container">
+        <article class="card card--width" data-qa="card">
+          <img
+            class="card__photo"
+            src="./images/imac.jpeg"
+            alt="Imac Monoblock photo"
+          >
+          <h1 class="card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rating">
+            <div class="stars stars__star--0">
```

Comment:
```suggestion
            <div class="stars stars--0">
```
Modifier should be added to the block, not the element